src/GenFacades/GenFacades.Core/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "Microsoft.Cci": "4.0.0-rc3-24214-00",
+    "Microsoft.DiaSymReader.Converter": "1.1.0-beta1-62225-01",
```

Comment:
Where does this dependency come from? We will ultimately need to be able to build it from source for our build-from-source efforts.

Reply:
It's in symreader-portable (https://github.com/dotnet/symreader-portable/tree/0b38450a75c9ea28b40020ac9b1a14a092ed1621/src/Microsoft.DiaSymReader.Converter) which we already have a submodule of in source-build.

Reply:
@weshaggard  also note that Microsoft.DotNet.Build.Tasks.dll (which is also build in BuildTools), already has a dependency on Microsoft.DiaSymReader.Converter.  Thus there is no really new dependency here.